client/management/v3/zz_generated_active_directory_config.go

```diff
@@ -75,4 +78,5 @@ type ActiveDirectoryConfig struct {
 	UserObjectClass              string            `json:"userObjectClass,omitempty" yaml:"userObjectClass,omitempty"`
 	UserSearchAttribute          string            `json:"userSearchAttribute,omitempty" yaml:"userSearchAttribute,omitempty"`
 	UserSearchBase               string            `json:"userSearchBase,omitempty" yaml:"userSearchBase,omitempty"`
+	UserSearchFilter             string            `json:"userSearchFilter,omitempty" yaml:"userSearchFilter,omitempty"`
```

Comment:
Should we go with 'additionalUserSearchFilter` instead - as per @cjellick's comment?